diff --git a/dist/index.mjs b/dist/index.mjs
index 4d4e696f5b681bd7256ce0a7e0b02c4ddef0ae11..f58e5907e5e17b4a63f6059bcbfb9d19a881950e 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -26,7 +26,11 @@ function codspeedPlugin() {
           pool: "forks",
           poolOptions: {
             forks: {
-              execArgv: getV8Flags()
+              execArgv: [
+                ...getV8Flags(),
+                '--no-concurrent-sweeping',
+                '--max-old-space-size=4096',
+              ]
             }
           },
           runner: getCodSpeedFileFromName("runner"),
diff --git a/dist/runner.mjs b/dist/runner.mjs
index f4c6c95e8e65391c140168e81ff553a199a395f2..36acc186a81ad8bd792c55417d804effa66bdf8b 100644
--- a/dist/runner.mjs
+++ b/dist/runner.mjs
@@ -5,15 +5,15 @@ import { NodeBenchmarkRunner } from 'vitest/runners';
 import { getHooks, getBenchFn } from 'vitest/suite';
 
 function getSuiteHooks(suite, name) {
-  return getHooks(suite)[name];
+  return getHooks(suite)?.[name] ?? [];
 }
 async function callSuiteHook(suite, currentTask, name) {
-  if (name === "beforeEach" && suite.suite) {
+  if (name === "beforeEach" && suite && suite.suite) {
     await callSuiteHook(suite.suite, currentTask, name);
   }
   const hooks = getSuiteHooks(suite, name);
   await Promise.all(hooks.map((fn) => fn()));
-  if (name === "afterEach" && suite.suite) {
+  if (name === "afterEach" && suite && suite.suite) {
     await callSuiteHook(suite.suite, currentTask, name);
   }
 }
